internal/plugin_test.go

```diff
@@ -1760,6 +1762,162 @@ spec:
 	assertEqual(t, output, expected)
 }
 
+func TestOverrideConstraintEnforcementAction(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	gatekeeperPath := path.Join(tmpDir, "gatekeeper.yaml")
+	yamlContent := `
+apiVersion: constraints.gatekeeper.sh/v1
+kind: MyConstrainingTemplate
+metadata:
+  name: thisthingimconstraining
+`
+
+	common := `
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/description: ""
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: policy-gatekeeper
+    namespace: gatekeeper-policies
+spec:
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: constraints.gatekeeper.sh/v1
+            kind: MyConstrainingTemplate
+            metadata:
+                annotations:
+                    policy.open-cluster-management.io/severity: low
+                name: thisthingimconstraining
+`
+
+	err := os.WriteFile(gatekeeperPath, []byte(yamlContent), 0o666)
+	if err != nil {
+		t.Fatalf("Failed to write %s", gatekeeperPath)
+	}
+
+	tests := []struct {
+		policyConf      types.PolicyConfig
+		policyDefaultEA string
+		expectedEA      string
+	}{
+		{
+			policyConf: types.PolicyConfig{
+				Name: "policy-gatekeeper",
+				Manifests: []types.Manifest{
+					{
+						Path:              path.Join(tmpDir, "gatekeeper.yaml"),
```

Comment:
Could you reuse `gatekeeperPath` here and also in other places?

Reply:
Good!